ds3-autogen-java/src/main/java/com/spectralogic/ds3autogen/java/converters/RequestConverter.java

```diff
@@ -80,21 +76,43 @@ private static String requestPath(final Ds3Request ds3Request) {
                 }
             }
         } else {
-            builder.append("_rest_/");
+            if (ds3Request.getResource() == Resource.BUCKET) {
```

Comment:
It looks like we will have to add this kind of logic for every resource type.  Maybe this is something that we can ask from jason.